lib/tests/2wp.js

```diff
@@ -105,13 +106,54 @@ const execute = (description, getRskHost) => {
 
     });
 
+    it('should do a basic pegin v1 with the exact minimum value', async () => {
+
+      // Arrange
+  
+      const initial2wpBalances = await get2wpBalances(rskTxHelper, btcTxHelper);
+      const senderRecipientInfo = await createSenderRecipientInfo(rskTxHelper, btcTxHelper);
+      const initialSenderAddressBalanceInSatoshis = await getBtcAddressBalanceInSatoshis(btcTxHelper, senderRecipientInfo.btcSenderAddressInfo.address);
+      const peginValueInSatoshis = minimumPeginValueInSatoshis;
+      const peginV1RskRecipientAddress = await rskTxHelper.newAccountWithSeed('');
+  
+      // Act
+  
+      const peginV1Data = [Buffer.from(createPeginV1TxData(peginV1RskRecipientAddress), 'hex')];
+  
+      const btcPeginTxHash = await sendPegin(rskTxHelper, btcTxHelper, senderRecipientInfo.btcSenderAddressInfo, Number(satoshisToBtc(peginValueInSatoshis)), peginV1Data);
+  
+      // Assert
+  
+      const expectedCountOfThisPeginUtxosInTheBridge = 1;
+      await ensurePeginIsRegistered(rskTxHelper, btcPeginTxHash, expectedCountOfThisPeginUtxosInTheBridge);
+  
+      const expectedPeginProtocolVersion = '1';
```

Comment:
Same here

Reply:
It defaults to '0', at the `createExpectedPeginBtcEvent` level.